subprojects/docs/src/docs/userguide/working_with_files.adoc

```diff
@@ -175,13 +175,22 @@ include::sample[dir="userguide/files/archives/kotlin",files="build.gradle.kts[ta
 
 As with a normal copy, you can control which files are unpacked via <<#sec:filtering_files,filters>> and even <<#sec:renaming_files,rename files>> as they are unpacked.
 
-More advanced processing can be handled by the link:{groovyDslPath}/org.gradle.api.tasks.AbstractCopyTask#eachFile(org.gradle.api.Action)[eachFile()] method. For example, you might need to extract different subtrees of the archive into different paths within the destination directory. The following sample uses the method to extract the files within the archive's `libs` directory into the root destination directory, rather than into `$destDir/libs`:
+More advanced processing can be handled by the link:{groovyDslPath}/org.gradle.api.tasks.AbstractCopyTask#eachFile(org.gradle.api.Action)[eachFile()] method. For example, you might need to extract different subtrees of the archive into different paths within the destination directory. The following sample uses the method to extract the files within the archive's `libs` directory into the root destination directory, rather than into a `libs` subdirectory:
 
 .Unpacking a subset of a ZIP file
 ====
 include::sample[dir="userguide/files/archives/groovy",files="build.gradle[tags=unpack-archive-subset-example]"]
 include::sample[dir="userguide/files/archives/kotlin",files="build.gradle.kts[tags=unpack-archive-subset-example]"]
 ====
+<1> Extracting only the files under `libs` directly into the destination directory
+<2> Remap the extracting files to a different path into the destination directory by dropping the `libs` segment from the file path
```

Comment:
Please rephrase the annotation text (or I can do it after a merge if you'd like). They should be worded based on asking yourself "what does this line do?" and then answering that question with "It <verb> ...". The "<verb> ..." is what goes into the annotation.

For example, what does line `<1>` do? It "extracts only the subset of files that reside in the `libs` directory.

As I said, I'm more than happy to modify these after you merge if you'd like.

Reply:
Sounds good. Let me try something and you can see if it's good.